tests/integration/llm/prepare.py

```diff
@@ -1114,6 +1114,12 @@
     "llama-3.1-8b": {
         "option.model_id": "s3://djl-llm/llama-3.1-8b-hf/",
     },
+    "llama-3.1-8b-multi-node-sharding": {
```

Comment:
How long does this take to shard ? Can we switch to the 1B model to save time ?

Reply:
It takes 8 min for sharding(including docker image downloading), 3 min for serving. Switching to 1b might only save us a few minutes

Reply:
Ack. No need to change if you have done the due diligence.